persistent/test/main.hs

```diff
@@ -900,8 +896,11 @@ takePrefix :: Value -> Value
 takePrefix (String a) = String (T.take 1 a)
 takePrefix a = a
 
-asTokens :: [T.Text] -> [Token]
-asTokens = fmap Token
+nonEmptyOrFail :: [a] -> NonEmpty a
+nonEmptyOrFail = maybe failure id . NEL.nonEmpty
```

Comment:
there's the horribly named`NEL.fromList :: [a] -> NonEmpty a` unless by some act of grace it was removed

Reply:
buuut if we just use list literal syyntax I think this can go away